pb/edge_ctrl_pb/edge_ctrl.proto

```diff
@@ -75,6 +78,7 @@ message ApiSessionUpdated {
 
 message ApiSessionRemoved {
     repeated string tokens = 1;
+    repeated string ids = 2;
```

Comment:
Added this id field to ApiSessionRemoved and SessionRemoved to enhance logging and make it easier to look up values in router logs. Prior to this only tokens were output which were hard to look up via the API. These are corresponding arrays and not tuples for backwards compact. Old controller + new router won't output the id information. New controller and older router, the id field will be ignored.